### PR TITLE
Tag GenericSVD v0.0.2

### PR DESCRIPTION
**Pull request by simonbyrne**
https://github.com/simonbyrne/GenericSVD.jl

### CODE REVIEW

File: GenericSVD/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline GenericSVD/versions/0.0.2/requires]: should have a minimum version for `view`

Comment (simonbyrne) [inline GenericSVD/versions/0.0.2/requires]: Is there an easy way to find it?

Comment (tkelman) [inline GenericSVD/versions/0.0.2/requires]: blame on the Compat.jl README works well

Comment (simonbyrne) [inline GenericSVD/versions/0.0.2/requires]: How can I convert that to a version number?

Comment (tkelman) [inline GenericSVD/versions/0.0.2/requires]: if you click on the commit, and the git tag for the Compat.jl release matched the metadata tagged sha and was pushed, you should get a little icon under the commit message that tells you which tags include the commit

**Pull request closed without merging**